packages/formik/src/Formik.tsx

```diff
@@ -489,7 +489,7 @@ export function useFormik<Values extends FormikValues = FormikValues>({
       isFunction(fieldRegistry.current[name].validate)
     ) {
       const value = getIn(state.values, name);
-      const maybePromise = fieldRegistry.current[name].validate(value);
+      const maybePromise = fieldRegistry.current[name].validate?.(value);
```

Comment:
this shouldn't be necessary, but I checked and for some reason the previous isFunction check should be enough to make typescript sure that value is a fn 👍